<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidget</class>
 <widget class="QWidget" name="VideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="renderLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="takePhotoButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Resim Al</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startRecordButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kayıt Başlat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopRecordButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kayıt Durdur</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateRecordLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>Record State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kamera Aç</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>takePhotoButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoWidget</receiver>
   <slot>takePhotoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoWidget</receiver>
   <slot>startRecordClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopRecordButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoWidget</receiver>
   <slot>stopRecordClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoWidget</receiver>
   <slot>startClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>takePhotoClick()</slot>
  <slot>startRecordClick()</slot>
  <slot>stopRecordClick()</slot>
  <slot>startClick()</slot>
 </slots>
</ui>
